Library/Homebrew/download_strategy.rb

```diff
@@ -1200,6 +1200,43 @@ def update
   end
 end
 
+# GCSDownloadStrategy downloads files from Google Cloud Storage
+# To use it, add `:using => :gs` to the URL section of your formula.
+# This strategy requires a GCS "project id" and a "credentials file",
+# information available here
+# https://cloud.google.com/storage/docs/authentication
+# You'll need to set the following environment variables in order for this
+# strategy to work properly:
+#  export HOMEBREW_GCS_PROJECT=<your google project id>
```

Comment:
⚠️ What happens if someone needs to access multiple projects?

Reply:
Google cloud differs from aws in this respect (at least it has been my experience) in that a user has to change project manually something like

```shell
gcloud config set project <project-name>
```

Let me see read through their docs to see  if there's some setting or env var that can be sent behind the scenes or something to fix this

Reply:
> To change the active configuration for all commands in your current terminal, you can set the environment variable CLOUDSDK_ACTIVE_CONFIG_NAME to the name of the configuration you'd like to use.

This requires the user to setup multiple configurations but if they're using multiple projects then that shouldn't be an issue I think.

I'll make the necessary changes